src/components/Text/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import BaseText from './BaseText';
+import {defaultProps, propTypes} from './baseTextPropTypes';
+
+class Text extends React.PureComponent {
```

Comment:
please rewrite to functional component, pure component (memoization) also is not needed